crates/hc/src/lib.rs

```diff
@@ -165,6 +166,8 @@ pub enum Opt {
     WebApp(hc_bundle::HcWebAppBundle),
     /// Work with sandboxed environments for testing and development
     Sandbox(hc_sandbox::HcSandbox),
+    /// Run a signal server
+    SignalSrv(hc_signal_srv::HcSignalSrv),
```

Comment:
`hc signal-srv` is not a great look IMO. I only know what a signal server is because I've followed your WebRTC work somewhat, but if I didn't, I don't know if I'd know how to interpret "signal-srv". I prefer `hc signal-server` but I wouldn't block this on the naming.

Structopt allows users to type the shortest unambiguous prefix, so for those who love brevity they can still just type `hc sig` either way.

Btw, searching for "signal server", the main hits are about the Signal messenger protocol, so it's probably good to qualify this as "WebRTC signal server" whenever possible in high visibility places where new users might engage with it.

[edit] searching for "webrtc signal server" shows that it's actually officially called a "signaling server". So :shrug:

Reply:
I would've preferred something else than `signal-srv` too! =D Sounds like "signal surf"!

Reply:
@jost-s @maackle - feel free to change it. Rust embraces short-cut names (`fn`, etc), I feel like we should have shortened it further to just `hc sig` lol : ), but I have zero attachment to the actual name.

Reply:
How about `hc ss`? :D

I'm not fond of the abbreviated names in Rust and was surprised to find them to be common. Long identifiers are cheap with code completion and compilation to byte code. For me it's more a general thing with that than specific to this case. If you know about webrtc you'll intuit signal-srv, and if you don't you probably won't. But signal-server doesn't explain you what it does either. So it should be `hc wss` for webrtc signal server ; )

Reply:
'course wss means Web-Sockets-Secure : )

Reply:
Yeah, it was purely meant as a joke.